### ISSUE TITLE
Tokenscan Dogeparty asset icons erroneously applied from Counterparty collections

### ISSUE THREAD

**Issue by 4x5x** [jdogresorg/freewallet-desktop#213]
Icons for assets in "Green Bar Counterparty Collections & Stamps" are erroneously being applied to assets of the same name on Dogeparty.

Started occuring after the migration to Tokenscan.

Rare Pepe Examples: RAREPEPE, PEPELIBRARY, PEPEMINING, SHITCOINCARD, GOXPEPE, PEPELEPEW
Spells of Genesis Example: FDCARD
Fake Rares Example: NORMIES
Counterparty BitBowl Example: PANTHERS, STEELERS
Bitcorns Example: BILLYMAIZE
Stamps Examples: BHAGAVADGITA, EIFFEL
ORD Example: RAREJORDAN

**Comment by CryptoEconomize** [comment]
Today my DOGECOINCARD vanished from my Freewallet balance.  Still shows in balance in other apps connected to that address but for some reason I can't find it in Freewallet anymore as of this morning.  I wondered if it had something to do with this ticket.

**Comment by 4x5x** [comment]
No, the issue you're experiencing is not related to this ticket.

Currently, Tokenscan and Freewallet which both operate on the same API are undergoing an upgrade. Per the blue banner at the top of the Tokenscan page:

"This site is currently undergoing an upgrade and will be updated when the upgrade is completed. Thank you for your patience!"

That means balances on Tokenscan and Freewallet could be incorrect until the update is completed.
